brayns/parameters/AnimationParameters.h

```diff
@@ -35,37 +36,59 @@ class AnimationParameters : public AbstractParameters
     /** @copydoc AbstractParameters::print */
     void print() final;
 
-    uint32_t getFrame() const { return _current; }
+    /** Reset to a 'no animation' state: 0 for dt, start and end. */
+    void reset();
+
+    /** The current frame number of the animation. */
     void setFrame(uint32_t value)
     {
-        _updateValue(_current, adjustCurrent(value));
+        _updateValue(_current, _adjustedCurrent(value));
     }
-
+    uint32_t getFrame() const { return _current; }
     /** The (frame) delta to apply for animations to select the next frame. */
     void setDelta(const int32_t delta) { _updateValue(_delta, delta); }
     int32_t getDelta() const { return _delta; }
+    uint32_t getStart() const { return _start; }
     void setEnd(const uint32_t end)
     {
         _updateValue(_end, end);
         _updateValue(_current, std::min(_current, _end));
     }
     uint32_t getEnd() const { return _end; }
-    void reset()
+    /** The dt of a simulation. */
+    void setDt(const double dt) { _updateValue(_dt, dt); }
+    double getDt() const { return _dt; }
+    /** The time unit of a simulation. */
+    void setUnit(const std::string& unit) { _updateValue(_unit, unit); }
+    using IsReadyCallback = std::function<bool()>;
+    using IsReadyCallbacks = std::list<IsReadyCallback>;
```

Comment:
```suggestion
    using IsReadyCallbacks = std::list<IsReadyCallback>;  // NOTE: Needs to be list for iterator validity
```

Reply:
Isn't it obvious? why else would one use a list :)

Reply:
Some oblivious coder might change it not realizing the consequences ;)